.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
 branches:
   only:
     - main
+    - toolkit-mvb
```

Comment:
we probably no longer need this as the branch will be deleted after merging
CC: @benbcai , @xenoworf

Reply:
Yes, we should leave his out of the main branch.

Reply:
removed : [2c845b](https://github.com/cerner/terra-toolkit/pull/742/commits/2c845bf7ed693bc89065fc790213087e2fc27230#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485)